src/Feather.jl

```diff
@@ -341,16 +352,16 @@ function writecolumn{T<:Union{Vector{UInt8},AbstractString}}(io, ::Type{T}, arr)
 end
 
 writenulls(io, A, null_count, len, total_bytes) = return total_bytes
-function writenulls(io, A::NullableVector, null_count, len, total_bytes)
+function writenulls(io, A::Vector{Union{T, Null}}, null_count, len, total_bytes) where {T}
     # write out null bitmask
     if null_count > 0
         null_bytes = Feather.bytes_for_bits(len)
-        bytes = BitArray(map(!, A.isnull))
+        bytes = BitArray(Bool[!isnull(x) for x in A])
```

Comment:
Why not  `isnull.(A)`? Iit creates a `BitArray` directly. Same for `CategoricalArray` below, with `broadcast(!iszero, A.refs)`.